cli/install_ee/install_ee.go

```diff
@@ -168,3 +168,40 @@ func FetchVersions(cliOpts *config.CliOpts) ([]version.Version, error) {
 
 	return versions, nil
 }
+
+func GetTarantoolEE(cliOpts *config.CliOpts, version string, dst string) error {
+	credentials, err := getCreds(cliOpts)
+	if err != nil {
+		return err
+	}
+	bundleName := "tarantool-enterprise-bundle-" + version + ".tar.gz"
+	eeLink := "https://download.tarantool.io/enterprise/" + bundleName
+	client := http.Client{Timeout: 5 * time.Second}
+	req, err := http.NewRequest(http.MethodGet, eeLink, http.NoBody)
+	if err != nil {
+		return err
+	}
+
+	req.SetBasicAuth(credentials.username, credentials.password)
+
+	res, err := client.Do(req)
```

Comment:
I don't think we should do any requests if dst directory does not exist. We will fail on os.create.

Reply:
added check